### PR TITLE
Review HAWKULAR-67 implementation

### PR DESCRIPTION
**Pull request by cardosogabriel**

### CODE REVIEW

**Comment by cardosogabriel** [comment]
Well done Juca! I comment inline:

- Dropdown switch: please follow the design specified in hawkular-67 + Manage Organizations link
- Spinner loading: use the same approach as Application Servers, for example. The title of the page should appear also only after the spinner.
- Create organization: the feedback (toastr) should be success and text: “Organization successfully created.”
- Organizations table: since we have a big space, we could list the members as hyperlinks, like:
	You (admin), Juraci, Thomas and 1 more 
	Clicking on one name would go to edit user. Clicking on 1 more would go to the users list screen.
- Organization table: tooltip Delete is not showing on the trash icon. Need fix.
- When removing an organization, we should have the confirmation dialog like in the urls list. Also, the feedback (toastr) should be success type (not info), and text: “Organization successfully deleted.”
- Organization not found: no need for feedback tostr on this page.

**Comment by jpkrohling** [comment]
Are the changes to the metrics less file intentional? Wouldn't they influence the look and feel of metrics? Apart from that, I'm ok with this PR.

File: console/src/main/scripts/plugins/metrics/less/metrics.less [hunk 1]
```diff
@@ -235,7 +251,7 @@ a:hover {
 
   .hk-graph-container  {
     margin: 0 auto;
-    //width: 750px;
+    width: 750px;
```
Comment (mtho11) [inline console/src/main/scripts/plugins/metrics/less/metrics.less]: Wouldn't this return us to fixed width graphs? And not the semi-responsive that we have now?

**Comment by cardosogabriel** [comment]
Yes, they are intentional :)

**Comment by mtho11** [comment]
@cardosogabriel looks like this needs to be rebased to master

**Comment by mtho11** [comment]
Merged #512

**Pull request merged**